.circleci/config.yml

```diff
@@ -221,6 +221,14 @@ generate-nightly-cache: &generate-nightly-cache
         mkdir -p ~/.pride-nightly/cache
         cp ~/.pride-nightly/last-commit ~/.pride-nightly/cache/
 
+# fix to address build issue: https://github.com/react-community/react-native-maps/issues/2159
```

Comment:
Shouldn't this be a yarn post install script? Will all developers need to run this command?

Also did you see these instructions that were linked to lower down in the issue? https://gist.github.com/theonetheycallneo/8d1d324a9e4bea30756da98c8c8ef617

Reply:
Yes, those were the instructions I used to install in the first place. I will double check the AirMaps portion of the instructions again.

No idea about the yarn post install stuff. This is over my head

Reply:
It would mean moving the sed command to a shell script. We would then be able to run that script in the post install stage of yarn. This means whenever a developer/ci runs `yarn install` the sed command will be run for them. We already have one command in the post install stage in our `package.json` so you'd have to pipe `||` then run your new script `./my-script-that-runs-sed.sh`. So it would look like `"postinstall": "yarn fix-xcode || ./scripts/my-script-that-runs-sed.sh"`

Reply:
Okay cool, I'll have a look at doing that. I went over the installation instructions again and thought it was all good but failed again when I pushed, sadly 🤷‍♀️

Reply:
Do we still need this, now that it is a `postinstall` script.

Reply:
I'm afraid I can't pretend to understand what's really happening here, or what it means to run this as a `postinstall` script as opposed to a build job or what the ramifications of removing it would be

Reply:
Now whenever the CI jobs runs yarn install the post install step will be run as well. This means the CI will already have the fix applied so this anchor isn’t needed anymore.